File: 344_Reverse_String.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: str
        :rtype: str
        """
        begin = 0
        end = len(s)-1
        strlist = [i for i in s]
        while(begin < end):
            camp =  strlist[begin]
            strlist[begin] = strlist[end]
            strlist[end] = camp
            begin +=1
            end -=1
        return ''.join(strlist)
        

File: 344_Reverse_String.cpp
class Solution {
public:
    string reverseString(string s) {
        int t = s.length();
        char a;
        for(int i = 0; i < t/2; i++){
            a = s[i];
            s[i] = s[t-1-i];
            s[t-1-i] = a;
        }
        return s;
    }
};
